index.js

```diff
@@ -35,7 +35,7 @@ const runBsbSync = () => {
 const getBsbErrorMessages = err => err.match(/File [\s\S]*?:\nError: [\s\S]*?\n/g)
 
 const getCompiledFile = (moduleDir, path, callback) => {
-  runBsb((err, res) => {
+  runBsb((err, _, res) => {
```

Comment:
As the doc https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback shows, `callback`'s parameters are `(error, stdout, stderr)`, we only care the `stderr` for emitting error.